tests/locale/keymap_or_locale.pm

```diff
@@ -34,7 +33,7 @@ sub run {
     return $self->verify_default_keymap_textmode_non_us($keystrokes, "${expected}_keymap") if ($expected ne 'us');
     $self->verify_default_keymap_textmode($keystrokes, "${expected}_keymap");
     $self->verify_default_keymap_textmode($keystrokes, "${expected}_keymap", console => 'root-console');
-    ensure_serialdev_permissions;
+    # ensure_serialdev_permissions is not needed as it is executing by system_prepare
```

Comment:
I guess you can remove this comment completely as the import has been removed too

Reply:
i left a comment on the commit message. but i was afraid that someone will wonder why it was removed. i just want to make sure if this is a request or suggestion from your side. and i can proceed.

Reply:
In such a case, I suggest keeping it as it was.